CONTRIBUTING.md

```diff
@@ -436,13 +436,35 @@ and follow below steps to add a rule for azure rest api specs.
 - add a doc for the rule in 'docs', the rule doc file name is following kebab-case, contains following content:
 
 ```md
-# RuleName
+# <add RuleName>
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+-<add RPC code, example: "-RPC-Uri-V1-04">
 
 ## Description
 
 <add the description for the rule.>
+
 ## How to fix
+
 <describe how to fix the violations.>
+
+## Good Examples
```

Comment:
I see a lot of existing .md files have "output message" section too. Do we plan to remove it?

Reply:
Yes, output message is same as error message so felt redundant adding in both the places.